src/V3AssertProperties.cpp

```diff
@@ -0,0 +1,280 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Transform sequences into properties
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2005-2025 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+//  Properties steps:
+//      Ensemble a property decision tree from sequence expressions.
+//          Property decision tree is a tree rooted with source sequence
+//          expression. It has a bipartite structure
+//          (Expr -> Stmt -> Expr ...).
+//      Transform property decision tree into AST, remove source sequence
+//      expression
+//          Each property is wrapped with AstPExpr that is transformed
+//          further by V3AssertPre and V3Assert.
+//
+//*************************************************************************
+
+#include "V3PchAstNoMT.h"  // VL_MT_DISABLED_CODE_UNIT
+
+#include "V3AssertProperties.h"
+
+#include "V3Graph.h"
+#include "V3UniqueNames.h"
+
+VL_DEFINE_DEBUG_FUNCTIONS;
+
+namespace {
+
+//######################################################################
+// Data structures (graph types)
+
+class DfaVertex VL_NOT_FINAL : public V3GraphVertex {
+    VL_RTTI_IMPL(DfaVertex, V3GraphVertex)
+    // STATE
+    AstNode* const m_nodep;  // Underlying node
+
+public:
+    // CONSTRUCTORS
+    explicit DfaVertex(V3Graph* graphp, AstNode* nodep) VL_MT_DISABLED : V3GraphVertex{graphp},
+                                                                         m_nodep{nodep} {}
+    AstNode* nodep() const { return m_nodep; }
+    // LCOV_EXCL_START // Debug code
```

Comment:
Remove.  Just e.g. add a '--dump-treei 9' to some appropriate test so these get covered.